spec/support/shared_examples_for_providers.rb

```diff
@@ -32,7 +32,10 @@
 shared_examples_for 'Rollbar integrated error handler' do
   context 'uncaught in a controller' do
     it 'reports the error to Rollbar' do
-      Rollbar.should_receive(:log).with('error', instance_of(RuntimeError)).and_return({})
+      Rollbar.should_receive(:log) do |level, args|
+        expect(level).to eq('error')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.